qa/pages/SiteMapPage.tsx

```diff
@@ -0,0 +1,139 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { GetServerSidePropsContext, InferGetServerSidePropsType } from 'next'
+import Link from 'next/link'
+import { NextSeo } from 'next-seo'
+import { ReactElement, useCallback } from 'react'
+import { FieldError, useForm } from 'react-hook-form'
+
+import { Container } from '@/components/Container/Container'
+import { RootLayout } from '@/components/Layout/RootLayout'
+import { TEXTAREA_MAX_CHARACTERS } from '@/constants/forms'
+import { useFormErrorHydration } from '@/hooks/useFormErrorHydration'
+import { logger } from '@/lib/logger'
+import { getValuesFromSearchParams } from '@/utils/form.utils'
+import { getCookieBanner } from '@/utils/getCookieBanner'
+import { FeedbackInputs, feedbackSchema } from '@/utils/schemas/feedback.schema'
+
+export type FeedbackProps = InferGetServerSidePropsType<typeof getServerSideProps>
+
+export default function Feedback({ query }: FeedbackProps) {
```

Comment:
@RahatH07 I see you've started the page by copying the Feedback page, which is fine, and this is a good page to copy from. However there is a lot of references to the original feedback page, with a number of Feedback page specific functions that we do not need

Reply:
you've also created your sitemap page inside /src/qa folder.
create it with the same directory naming as src e.g.
src/pages/sitemap/index.tsx